packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -1,9 +1,13 @@
+import { PropertyKeys } from './membrane';
 import { VirtualEnvironment } from './environment';
 
 const { includes: ArrayProtoIncludes, push: ArrayProtoPush } = Array.prototype;
 const { assign: ObjectAssign } = Object;
 const { apply: ReflectApply, ownKeys: ReflectOwnKeys } = Reflect;
 
+// @ts-ignore
+export const SUPPORTS_SHADOW_REALM = typeof ShadowRealm === 'function';
```

Comment:
should this be in the DOM package directly? why in base? I don't think it is used in Base.